### ISSUE TITLE
Here is Solution for gem installation errors after installing beef

### ISSUE THREAD

**Issue by Hack-KingDav** [beefproject/beef#2953]
1. After downloading see the installation process here: https://github.com/beefproject/beef/wiki/Installation

2. Configure the user_name and Password from config.yaml file.

3. After executing the command "./beef if you got  some errors about gem bundles installation, like: 

"  Could not find gem 'test-unit-full (~> 0.0.5)' in locally installed gems.
Run `bundle install` to install missing gems.   "

then for all this type of errors, simply run command as root user "gem install BUNDLE_NAME_GIVEN_IN_ERROR".

The most of them are here to save your installation time: 

(1).  sudo gem install xmlrpc &&  gem install unf

(2).  sudo gem install rushover

(3).  sudo gem install test-unit-full

(4).  sudo gem install rspec

(5).  sudo gem install rdoc

(6).  sudo bundle install

(7).  sudo gem install domain_name

(8).  sudo gem install http-cookie

(9).  sudo bundle install

Keep Executing these commands one-by-one for a frustration-free installation of the 'beef'. 😎 👍

(10). If you want to do some more Configurations, go here: https://github.com/beefproject/beef/wiki/Configuration

                                                                                    ** Respect Privacy of Everyone **

(11).                                                                                     Now Good To Go 👍

**Comment by stephenakq** [comment]
Thank you @Hack-KingDav. What was the main challenge?

**Comment by Hack-KingDav** [comment]
Hello @stephenakq , sorry for late response. I appreciate you replied, being a contributor to the Project. The main Problem with me was that it was echoing errors on console screen about some of the required packages, essential for the Software-Patch to Use efficiently. So I decided to make a report on troubleshooting these errors directly through a single Report, for users Conveniency.

**Comment by stephenakq** [comment]
Thank you. This might be best in the wiki

**Comment by andyzot** [comment]
Hi, I appreciate this has been marked as closed but I'm struggling to find a solution which bears a resemblance to this.
I indeed had to manually install the gem files one but one, as each one was successfully installed, I moved to the next and tried to run beef.
I got to a stage where I had to install 'websocket-client'simple' and it said it was successfully installed, but when I try to run beef I get the message -
"Could not find gem 'websocket-client-simple (~> 0.6.1)' in locally installed gems.
                                                                                                                   
The source contains the following gems matching 'websocket-client-simple':                                         
  * websocket-client-simple-0.9.0                                                                                  
Run `bundle install` to install missing gems."

This just repeats, any help appreciated.
